app/javascript/widget/App.vue

```diff
@@ -86,7 +87,7 @@ export default {
     },
     setLocale(locale) {
       const { enabledLanguages } = window.chatwootWebChannel;
-      if (enabledLanguages.some(lang => lang.iso_639_1_code === locale)) {
+      if (enabledLanguages.some((lang) => lang.iso_639_1_code === locale)) {
```

Comment:
There are some syntax changes with parenthesis of arrow functions, which is not aligned to other parts of code. Can you revert them ?